src/main/java/net/openhft/chronicle/wire/BinaryWireCode.java

```diff
@@ -35,7 +35,7 @@ public enum BinaryWireCode {
     // sequence of length 0 - 255
 //        public static final int BYTES_LENGTH64 = 0x83;
 
-    public static final int BYTES_MARSHALLABLE = 0x86; // explicitly BytesMarshallable.
+    public static final int HISTORY_MESSAGE = 0x86; // was BYTES_MARSHALLABLE, but only used for this purpose
```

Comment:
we should probably go back in time and mark `BYTES_MARSHALLABLE` as deprecated? Or are you confident enough that it is a internal or domestic API and won't be likely to be used outside of Chronicle products?